src/helpers/dom/element.js

```diff
@@ -762,11 +762,47 @@ export function getComputedStyle(element, rootWindow = window) {
 /**
  * Returns the element's outer width.
  *
+ * @see {preciseOuterWidth}
  * @param {HTMLElement} element An element to get the width from.
  * @returns {number} Element's outer width.
  */
 export function outerWidth(element) {
-  return Math.ceil(element.getBoundingClientRect().width);
+  return element.offsetWidth;
```

Comment:
Shouldn't be there some kind of `ceil` or counted `scaleX` usage? I don't know, I just asking to be sure that it should be like this

Reply:
What do you mean by "counted `scaleX` usage"?

This function reverts the change made in https://github.com/handsontable/handsontable/commit/99416f29c5a418cfb670a32cbdd67b1f5e117f95, `preciseOuterWidth` works like the old `outerWidth`, but way slower.

Reply:
> What do you mean by "counted scaleX usage"?

I was wondering whether using cached variable from `preciseOuterWidth` would be needed.

> This function reverts the change made in 99416f2

Ah, yeah. Ok. I haven't checked how it works on Safari btw. I will do it.

Reply:
It looks like it works! Nice 👏